recipe/meta.yaml

```diff
@@ -40,6 +40,7 @@ source:
     - patches/0001-force-one-modulemap-for-darwin.patch  # [osx]
     - patches/0001-Remove-check-for-SDK-version-in-CIFactory.patch  # [osx]
     - patches/0001-avoid-linking-TPython-against-libPython.patch  # [osx]
+    - patches/0001-std_darwin.modulemap-Remove-headers-for-MacOSX15.0.s.patch  # [osx]
```

Comment:
Maybe drop the patch, reset the build number, ad skip CI?

That way this is just maintainer changes and CI is not really needed then